appsec/tests/integration/src/test/groovy/com/datadog/appsec/php/integration/CommonTests.groovy

```diff
@@ -238,6 +238,24 @@ trait CommonTests {
         assert span.meta."_dd.appsec.event_rules.version" != ''
     }
 
+    @Test
+    void 'user login fingerprint'() {
+        def trace = container.traceFromRequest('/user_login_success.php?id=user2020') { HttpResponse<InputStream> resp ->
+            assert resp.statusCode() == 403
+            assert resp.body().text.contains('blocked')
+        }
+
+        Span span = trace.first()
+        assert span.meta."appsec.blocked" == "true"
+        assert span.metrics."_dd.appsec.enabled" == 1.0d
+        assert span.metrics."_dd.appsec.waf.duration" > 0.0d
+        assert span.meta."_dd.appsec.event_rules.version" != ''
```

Comment:
I would remove this assertions. The reason is because somehow tests need to be also used as documentation. This test is focus on fingerprint so anything else which is not related to fingerprint, I would personally removed to keep the focus on what it needs to be. For example, the fact that this user is blocked or not, it does not changes anything on the fingerprints

Reply:
Yeah you are right, I just removed them 👌